include/program_options_utils.hpp

```diff
@@ -73,7 +73,9 @@ const char *LABEL_FILE = "Input label file in txt format for Filtered Index buil
 const char *UNIVERSAL_LABEL =
     "Universal label, Use only in conjunction with label file for filtered index build. If a "
     "graph node has all the labels against it, we can assign a special universal filter to the "
-    "point instead of comma separated filters for that point";
+    "point instead of comma separated filters for that point.  The universal label should be assigned to nodes "
+    "in the labels file instead of listing all labels for a node.  DiskANN will not automatically assign a "
```

Comment:
hi Jon, What is the goal of this PR? Is the current explanation unclear?

Reply:
To me the existing explanation was unclear.

> we can assign a special universal filter to the point instead of comma separated filters for that point

I had to look at the code to figure this out.  I thought that DiskANN would assign the universal filter automatically if the user specified all the available filters since the documentation said "we can assign".  My read of the code is that DiskANN does not automatically convert a complete filter set to the universal filter.

Reply:
Your read is correct. Sounds good. thanks!